build.bat

```diff
@@ -5,7 +5,7 @@ set __OLD_vcvars64=%vcvars64%
 set vcvars64="C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
 set CLICOLOR_FORCE=1
 
-set NINJAFLAGS=
+set NINJAFLAGS=-k0
```

Comment:
This is for debugging, you shouldn't commit it. You can comment on the PR as a test plan though.